cloudformation/bulkExport.yaml

```diff
@@ -77,3 +77,96 @@ Resources:
                 Action:
                   - glue:StartCrawler
                 Resource: !Join ['', [!Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:crawler/', !Ref Crawler]]
+
+  GlueJobRelatedLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: 'Allow'
+            Principal:
+              Service: 'lambda.amazonaws.com'
+            Action: 'sts:AssumeRole'
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+        - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
+      Policies:
+        - PolicyName: glueAccess
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - glue:StartJobRun
+                  - glue:GetJobRun
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - dynamodb:Query
+                  - dynamodb:GetItem
+                Resource:
+                  - !GetAtt ExportRequestDynamoDBTable.Arn
+
+  GlueScriptsBucket:
+    Type: AWS::S3::Bucket
+
+  BulkExportResultsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpirationRule
+            Status: Enabled
+            ExpirationInDays: '7'

```

Comment:
I think the items in the bucket only need to stay in the bucket for as long as the presigned Url for the exported filles are valid. It's unlikely the presigned url is going to last 7 days. 

Maybe we can set a custom variable value in serverless for how long the presigned S3 url is going to last. Provide that as an environment variable so we can generate a presigned S3 url for that given length of time. Then for the S3 Bucket policy have the expiration time equal to presigned S3 url expiration time + 15 minutes. We can exchange 15 minutes for an hour to be safe.

Reply:
Could they request another presigned URL for the same export file?

Reply:
The original tech design is we create the presigned s3 url once the export is complete. We store that in DDB and retrieve that when a user asks for the files. In this approach a new url is not generated.

Nestor and I discussed offline about allowing the url to be regenerated though. I believe Nestor is going to send an email to the group to discuss the new approach. And we can confirm with Angus/Steven that the new approach would still be ok for our customers.

Reply:
Changed to 3 days

Reply:
We are still having conversations about the exact details around presigned s3 urls. 
I may make an adjustment to this bucket lifecycle policy later on when working on the presigned urls story. It's probably fine to not block this PR on this detail.